app/browser/activeTabHistory.js

```diff
@@ -0,0 +1,57 @@
+// static
```

Comment:
can we put the standard preamble at the top of this file?

Reply:
done, and rebased